test/unittests/test_create_ontology.py

```diff
@@ -7,7 +7,9 @@
 
 
 class TestOntoCreation(unittest.TestCase):
-    ontology: dict[str, Any] = json.loads('testdata/test-onto.json')['project']['ontologies'][0]
+    with open('testdata/test-onto.json', 'r') as json_file:
+        json_onto = json.loads(json_file.read())
```

Comment:
I'm pretty sure you can use json.load() with just the file, rather than taking the detour with the string

```suggestion
        json_onto = json.load(json_file)
```

Reply:
You are right. I updated it.